src/nvim/eval.c

```diff
@@ -10608,6 +10608,9 @@ static void f_has(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     "fname_case",
 #ifdef HAVE_ACL
     "acl",
+#endif
+#ifdef FEAT_AUTOCHDIR
```

Comment:
Why is an `#ifdef` needed here?

Reply:
Ah damn, that shouldn't be there at all.